diff --git a/node_modules/@reown/appkit/dist/esm/src/networks/solana/index.js b/node_modules/@reown/appkit/dist/esm/src/networks/solana/index.js
index bffa431..aa67b33 100644
--- a/node_modules/@reown/appkit/dist/esm/src/networks/solana/index.js
+++ b/node_modules/@reown/appkit/dist/esm/src/networks/solana/index.js
@@ -1,4 +1,5 @@
 export * from './solana.js';
 export * from './solanaDevnet.js';
 export * from './solanaTestnet.js';
+export * from './solanaLocalnet.js';
 //# sourceMappingURL=index.js.map
\ No newline at end of file
diff --git a/node_modules/@reown/appkit/dist/esm/src/networks/solana/solanaLocalnet.js b/node_modules/@reown/appkit/dist/esm/src/networks/solana/solanaLocalnet.js
new file mode 100644
index 0000000..3f3d6d7
--- /dev/null
+++ b/node_modules/@reown/appkit/dist/esm/src/networks/solana/solanaLocalnet.js
@@ -0,0 +1,14 @@
+import { defineChain } from '../utils.js';
+export const solanaLocalnet = defineChain({
+    id: '11111111111111111111111111111111',
+    name: 'Solana Localnet',
+    network: 'solana-localnet',
+    nativeCurrency: { name: 'Solana', symbol: 'SOL', decimals: 9 },
+    rpcUrls: {
+        default: { http: ['http://127.0.0.1:8899'] }
+    },
+    blockExplorers: { default: { name: 'Solana Explorer', url: 'https://explorer.solana.com/?cluster=custom&customUrl=http%3A%2F%2F127.0.0.1%3A8899' } },
+    testnet: true,
+    chainNamespace: 'solana',
+    caipNetworkId: 'solana:11111111111111111111111111111111'
+});
\ No newline at end of file
